lib/travis/build/script/csharp.rb

```diff
@@ -321,10 +330,26 @@ def is_mono_after_5_0
         def is_dotnet_after_2_0?
           return false unless is_dotnet_version_valid?
 
-          return false if DOTNET_VERSION_REGEXP.match(config_dotnet)[1].to_i < 2
+          return false if DOTNET_VERSION_REGEXP.match(config_dotnet)[:major].to_i < 2
 
           true
         end
+
+        # Starting in 2.1.300, the package ID changed from dotnet-sdk-(major.minor.patch) to dotnet-sdk-(major.minor)
+        def is_dotnet_after_2_1_300?
```

Comment:
You can use `Gem::Version` to compare version numbers. See, for example, https://github.com/travis-ci/travis-build/blob/a79566b678464c80cb613f8636cbe89114d623a0/lib/travis/build/script/elixir.rb#L88-L91

Reply:
Awesome, that's much cleaner. Just updated to use that.